RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml.cs

```diff
@@ -22,11 +22,13 @@ private void TreeView_OnMouseDoubleClick(object sender, MouseButtonEventArgs e)
             {
                 ViewModel.NavigateCommand.Execute(ViewModel.SelectedItem);
             }
+            e.Handled = true;
         }
 
-        private void OnPreviewMouseRightButtonDown(object sender, MouseButtonEventArgs e)
+        private void TreeView_OnMouseRightButtonDown(object sender, MouseButtonEventArgs e)
```

Comment:
All mouse click events are in pairs: `PreviewMouse*Down` and `Mouse*Down`.  `Preview` events "tunnel" down the window from the outer layers, `Mouse` down events "bubble" up from the bottom of the view.  Hence, "PreviewMouseRightButtonDown" started at the top of the tree with the Project, and continued to work down to the actual selected node."  It only worked because `e.Handled = true;` didn't stop the events from working down the tree.

https://msdn.microsoft.com/en-us/library/system.windows.uielement.previewmouserightbuttondown(v=vs.110).aspx
https://msdn.microsoft.com/en-us/library/system.windows.uielement.mouserightbuttondown(v=vs.110).aspx